frontstage/templates/secure-messages-draft.html

```diff
@@ -46,8 +46,7 @@ <h1 class="saturn">Edit a draft message</h1>
                                 class="secure-message-form__subject input input--text"
                                 type="text"
                                 name="secure-message-subject"
-                                value="{{draft['subject']}}"
-                                required/>
+                                value="{{draft['subject']}}"/>
                         </fieldset>
```

Comment:
The pattern we follow is {{ draft['subject'] }} so spaces after {{ and before }}